fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/checkpoint/CheckPointAlgorithm.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * (C) Copyright IBM Corp. 2019, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.bulkdata.jbatch.export.checkpoint;
+
+import java.util.logging.Logger;
+
+import javax.batch.api.chunk.CheckpointAlgorithm;
+import javax.batch.runtime.context.StepContext;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import com.ibm.fhir.bulkdata.jbatch.export.data.TransientUserData;
+import com.ibm.fhir.operation.bulkdata.config.ConfigurationFactory;
+
+/**
+ * BulkData Export Custom CheckpointAlgorithm which considers COS size requirements while checkpointing.
+ */
+@Dependent
+public class CheckPointAlgorithm implements CheckpointAlgorithm {
```

Comment:
having the instance class name vary from the interface name only by casing is a little odd.  is there any distinguishing feature from other checkpoint algorithm which we can use in the name?  maybe `BulkExportCheckpointAlgorithm`?

Reply:
yes - totally agree.  I'll make that change